sqlcapture/capture.go

```diff
@@ -339,6 +339,26 @@ func (c *Capture) updateState(ctx context.Context) error {
 			return fmt.Errorf("stream %q: primary key must be specified", streamID)
 		}
 
+		// See if the stream is already initialized. If it's not, then create it.
+		var streamState, ok = c.State.Streams[streamID]
+		if !ok || streamState.Mode == TableModeIgnore {
+			c.State.Streams[streamID] = &TableState{Mode: TableModePending, KeyColumns: primaryKey, dirty: true}
+			continue
+		}
+
+		// The following safety checks only matter if we're actively backfilling the table or
+		// intend to start backfilling it shortly. But checking for that would risk silently
+		// failing if a new keyed backfill mode were added in the future, so instead we check
+		// for the set of known-to-be-fine situations here.
+		var notBackfilling = (streamState.Mode == TableModeIgnore)
+		notBackfilling = notBackfilling || (streamState.Mode == TableModeActive)
```

Comment:
nit:
```
var notBackfilling = (streamState.Mode == TableModeIgnore)
    || (streamState.Mode == TableModeActive)
    || (streamState.Mode == TableModeKeylessBackfill)
    ...
```

Reply:
Sadly Go doesn't let you line the operators up nicely like that and the best I can do is:

```
var notBackfilling = (streamState.Mode == TableModeIgnore) ||
	(streamState.Mode == TableModeActive) ||
	(streamState.Mode == TableModeKeylessBackfill) ||
	(streamState.Mode == TableModePending && binding.Resource.Mode == BackfillModeOnlyChanges) ||
	(streamState.Mode == TableModePending && binding.Resource.Mode == BackfillModeWithoutKey)
```